.github/actions/deploy-safe-tools-client/action.yml

```diff
@@ -0,0 +1,65 @@
+name: Deploy safe-tools-client
+description: Deploy safe-tools-client to S3 and Cloudfront
+
+inputs:
+  environment:
+    description: Deployment environment
+    required: true
+  discord_webhook:
+    description: Discord webhook URL
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Set up env
+      shell: bash
+      env:
+        INPUT_ENVIRONMENT: ${{ inputs.environment }}
+      run: |
+        if [ "$INPUT_ENVIRONMENT" = "production" ]; then
+          echo "AWS_ROLE_ARN=arn:aws:iam::120317779495:role/safe-tools-client" >> $GITHUB_ENV
+          echo "S3_BUCKET=cardstack-safe-tools-client-production" >> $GITHUB_ENV
+          echo "CLOUDFRONT_DISTRIBUTION=PLACEHOLDER" >> $GITHUB_ENV
+        elif [ "$INPUT_ENVIRONMENT" = "staging" ]; then
+          echo "AWS_ROLE_ARN=arn:aws:iam::680542703984:role/safe-tools-client" >> $GITHUB_ENV
+          echo "S3_BUCKET=cardstack-safe-tools-client-staging" >> $GITHUB_ENV
+          echo "CLOUDFRONT_DISTRIBUTION=PLACEHOLDER" >> $GITHUB_ENV
+        else
+          echo "unrecognized environment"
+          exit 1;
+        fi
+
+    - uses: volta-cli/action@v1
+
+    - run: yarn --prefer-offline --frozen-lockfile
+      shell: bash
+
+    - name: Build
+      shell: bash
+      working-directory: packages/safe-tools-client
+      run: yarn build:safe-tools-client --environment ${{ inputs.environment }}
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        role-to-assume: ${{ env.AWS_ROLE_ARN }}
+        aws-region: us-east-1
+
+    - name: Sync content to S3
```

Comment:
I would prefer to use ember-cli-deploy here. It is standard in the ember world and allows developers to add additional plugins to adjust the deploy process. Happy to help with this if you want.

Reply:
Added the ember deploy plugins and updated the action and script.
I am unfamiliar with the tools though, so definitely appreciate the help to review them thoroughly!